Dockerfile

```diff
@@ -80,10 +89,14 @@ RUN pip2 install --no-cache-dir mock nose requests testpath && \
     pip2 uninstall -y funcsigs mock nose pbr requests six testpath && \
     pip3 uninstall -y nose requests testpath
 
+# Add a notebook profile.
+RUN jupyter notebook --generate-config && \
```

Comment:
This isn't the way to do do this kind of thing. If we want a config file, it should be written in its entirety, either with `echo ... >> path` or `ADD`. There is no need to do generate-config and then modify in-place.